processing/src/main/java/org/apache/druid/frame/file/FrameFile.java

```diff
@@ -246,7 +246,8 @@ static FrameFile open(
             log.warn("Could not delete frame file [%s]", file);
           }
           if (byteTracker != null) {
-            byteTracker.release(fileLength);
+            // only release the bytes taken by frames, we don't track the header and footer as of now
```

Comment:
Just curious why was this change needed.

Reply:
this was needed since we're not tracking header and footer size as of now (due to difficulties with empty frame files). I have added an explanation for this in the code - it could be picked up in the future if needed.
Also, the assumption is that frame data would be much larger than the header + footer sizes.